### DISCUSSION TITLE
What major topics do you want us to discuss on Friday?

### DISCUSSION THREAD

**Discussion by palday** [RePsychLing/SMLP2023#39] [Q&A]
One topic per answer and upvote other answers that interest you!

**Comment by ansost** [comment]
Model Comparison

**Comment by yjunechoe** [comment]
What language to use when reporting (in publications) some of the strategies we discussed, especially:

- Profiling
- Bootstrapped confidence intervals

And how to cite MixedModels.jl :)

**Reply by palday** [reply]
There's a Zenodo link for citation on the [MixedModels homepage](https://github.com/JuliaStats/MixedModels.jl).

**Reply by palday** [reply]
> Confidence intervals were computed using the parametric bootstrap with n replicates.

where n is the number you used -- at the very least 1000, but preferably 5000 or even 10000.

You should also specify whether you used the equal-tail probability interval or the highest density (shortest) interval.

For profiling,

> Confidence intervals were computed by profiling of the likelihood surface.

**Comment by pandongpeng** [comment]
Reparameterization in post-hoc analysis with examples.

**Comment by yjunechoe** [comment]
(If there's time) A worked out example (or at least a list of things to consider) for parametricbootstrap with a **numerical covariate** too! (Edit for clarity: parametricbootstrap w.r.t. power analysis)

**Comment by TeichPaula** [comment]
Principal component analysis

**Reply by ansost** [reply]
Including the difference between `fit` and `predict`:) Most explanations I saw are for train and test data but I am not sure whether I would have to use both `fit` and `predict` or just `fit` if I dont split my data.

**Comment by pianobacki** [comment]
non linear covariates:
more examples on how to deal with covariates (like time), if this covariate shows a nonliniear behavior. 
e.g. ERP responses in 4 different tasks; 
how to center (meaningful zero), 
scaling?
how to transform? (inside the model or before in the dataset)
how to include in the model (only the transformed version, or also in original metric)
what can go wrong?

Examples were the dv has a non linear trend over  time:  
<img width="857" alt="Bildschirmfoto 2023-09-14 um 17 39 18" src="https://github.com/RePsychLing/SMLP2023/assets/32771790/da107945-28c6-4b86-89e6-00730dc16b0a">

<img width="868" alt="Bildschirmfoto 2023-09-14 um 18 59 35" src="https://github.com/RePsychLing/SMLP2023/assets/32771790/7a4ffa7a-7cf5-4e5b-b835-08215d95c107">

**Comment by cyhsieh-psy** [comment]
GLMM for different types of dependent variables (e.g. count data, proportion data, ordinal data)

**Comment by ebruevcen** [comment]
(If time allows) a worked out example of power analysis (maybe using one of the available datasets?), if such a thing is ever possible? If not, why? Also, how do I decide the values (e.g., β, σ) for a priori power simulation?
